### PR TITLE
Fix #10741: Call Out-File with an encoding for make-hosts-file on Windows

### LINKED ISSUE #10741: Python checks for hosts file are not working anymore
After @gsnedders recent changes to run.py, tests no longer work on Windows. During the check for the hosts file, it throws an error suggesting you should create the hosts entries. However, doing so does not help.

When the entries are created, there is an additional space character injected between each char

eg:  1 2 7 . 0 . 0 . 1 	 w e b - p l a t f o r m . t e s t

So then later, the checks fail. If I remove the spaces, then all is good.

I was just going to submit a new PR for that, but I cannot find the spot where those spaces are added. I'm still checking, though.

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes #10741.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10800)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ For example, on most UNIX-like systems, you can setup the hosts file with:
 ./wpt make-hosts-file | sudo tee -a /etc/hosts
 ```
 
-And on Windows (note this requires an Administrator privileged shell):
+And on Windows (note this requires in PowerShell with Administrator privileges):
```
Comment (jugglinmike) [inline README.md]: How about "this requires PowerShell with Administrator privileges" or "this must be run in PowerShell with Administrative privileges"?

**Review by jgraham — APPROVED** [review]
Would be nice to make it clear that using something other than PowerShell works like normal, otherwise fine modulo the typo

**Comment by gsnedders** [comment]
@jgraham I don't know many people still using `cmd.exe` on Windows; it's either Bash on Windows or PowerShell nowadays (and we didn't deal with the former before, given we'd need to generate a different path for it).

File: README.md (hunk 1 above)
Comment (gsnedders) [reply README.md]: I was trying to figure out how MS refer to it; I think something like the latter is probably what we want.

Comment (gsnedders) [reply README.md]: Does this now look better?

Comment (jugglinmike) [reply README.md]: Much!

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**